xml/admin_operating_services.xml

```diff
@@ -233,4 +233,127 @@ ceph-rbd-mirror@.service
 <screen>&prompt.smaster;salt-run state.orch ceph.restart.ganesha</screen>
   </sect2>
  </sect1>
+ <sect1 xml:id="ceph-cluster-shutdown">
+  <title>Shutdown and Start of the Whole &ceph; Cluster</title>
+
+  <para>
+   There are occasions when you need to stop all &ceph; related services in the
+   cluster in the recommended order, and then be able to simply start them
+   again. For example, in case of a planned power outage.
+  </para>
+
+  <procedure>
+   <title>Shutting Down the Whole &ceph; Cluster</title>
```

Comment:
Probably don't need this title, since it's the title of the section too

Reply:
I dont agree, there are 2 adjacent procedures and the reader should know which one does what, or no? One for shutdown and one for startup.

Reply:
Hmmm okay, you're right about that. Leave it.